lemarche/templates/siaes/_card_suggest_tender.html

```diff
@@ -1,13 +1,16 @@
 {% comment %} {% load static bootstrap4 %} {% endcomment %}
-
 <div class="c-box mb-4">
     <div class="row">
         <div class="col-md">
-            <span class="badge badge-sm badge-pill badge-warning-light mb-2">Information pratique</span>
-            <h2 class="h2 mb-2">Pas de temps à perdre ?</h1>
-            <p>Confiez-nous votre recherche (ou sourcing) et recevez des réponses de prestataires inclusifs en moins de 24h.</p>
-            <a href="{% url 'tenders:create' %}" id="siae-search-insert-tender-create-btn" class="btn btn-primary btn-sm btn-ico d-block d-md-inline-block mb-2">
-                <i class="ri-mail-send-line ri-lg"></i>&nbsp;Publier un besoin d'achat
+            <span class="badge badge-sm badge-pill badge-primary mb-2">Nouveauté</span>
```

Comment:
```suggestion
            <span class="badge badge-sm badge-pill badge-important mb-2">Nouveauté</span>
```
pour avoir la couleur violet similaire à l'onglet du haut ?

Reply:
bien vu !